### DISCUSSION TITLE
Any idea why 65.6kps upload appears to be a constant upload speed

### DISCUSSION THREAD

**Discussion by GaryElshaw** [transmission/transmission#4244] [General]
Something i've noticed recently, if the upload rate isn't fluctuating, 65.6k seems to be a constant upload rate.

**Comment by ckerr** [comment]
No idea, FWIW I'm seeing higher numbers. :shrug:

One possible theory is that you're seeding to a particular peer and that's how much bandwidth they have available.

AFAIK it's not a limitation in the code

**Reply by GaryElshaw** [reply]
I see it occasionally happening with different torrents and peers, i just thought it appeared to be such an odd value to be so consistent.
